libclamav/scanners.c

```diff
@@ -979,6 +979,190 @@ static cl_error_t cli_scanegg(cli_ctx *ctx)
     return status;
 }
 
+// static cl_error_t cli_scanonenote(cli_ctx *ctx)
```

Comment:
Do you still need this?

Reply:
Urgh.  I do not. Forgot all about it.  Will remove.